index.html

```diff
@@ -11,14 +11,34 @@
     </head>
     <body>
       
-          <!--Navbar-->
           <nav id="nav">
             <a class="logo" href="#">My Logo</a>
             
-            <nav><img src="./Images/Union.svg" alt="burger-menu"></nav>
+            <ul class="nav-links">
+              <!-- USING CHECKBOX HACK -->
+              <input type="checkbox" id="checkbox_toggle" />
+              <!-- <nav class="hamburger"><img src="./Images/Union.svg" alt="burger-menu"></nav> -->
+              <label for="checkbox_toggle" class="hamburger">&#9776;</label>
+              <!-- NAVIGATION MENUS -->
+              <div class="menu">
+                <li><a href="/">Portfolio</a></li>
+                <li><a href="/">About</a></li>
+                <li><a href="/">Contact</a></li>
+             
+        
                
           </nav>
```

Comment:
- [ ]  Kyndly stick to the figma template as much as you can,  please notice that in the figma template the navbar does not take the 100% of the width of the page, there are some spaces on the right and left side of the navbar. ✏️

Your design 

![image](https://user-images.githubusercontent.com/95243996/201193584-472d47aa-6da7-487f-8d7f-837f24e5dd19.png)

Figma template

![image](https://user-images.githubusercontent.com/95243996/201193995-4f66a2db-0dbb-49a2-97fb-4f4df308a923.png)